Dockerfile.slim

```diff
@@ -1,4 +1,4 @@
-FROM openjdk:8-jre-alpine
+FROM openjdk:8-jdk-alpine
```

Comment:
Is it possible to use `jre` in place of `jdk`? Reason I'm asking is this change makes the docker image quite a bit larger.

Reply:
Unfortunately, no - running `gcloud app deploy` for Java 8 Standard applications does rely on having a JDK available.

Reply:
Oh well, LGTM